wishlist-page/wishlist.html

```diff
@@ -0,0 +1,128 @@
+                    <li><a href="/cart-page/cart.html"><i class="fa fa-shopping-cart"></i><span class="items-quant">2</span></a></li>
+                </div>
+            </ul>
+        </nav>
+        <div class="wishlist-container">
+            <div class="wishlist-title">
+                <h2>My WishList : </h2>
+                <span class="text-secondary">3 Items</span>
+            </div>
+            <div class="wishlist-main-content">
+                <div class="wishlist-list">
+                    <div class="vertical-card simple-card">
+                        <img src="/assets/toy2.jpg" alt="image">
+                        <div class="card-desc-block">
+                            <h1>Mickey Mouse</h1>
+                            <p>Multicoloured soft toys Disney</p>
+                            <span class="text-secondary">₹299</span>
+                            <div class="rating-container">
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon fa fa-star"></i>
+                            </div>
+                            <br>
+                            <div class="card-buttons">
+                                <a class="cart-button">Add to Cart</a>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="vertical-card simple-card">
+                        <img src="/assets/toy2.jpg" alt="image">
+                        <div class="card-desc-block">
+                            <h1>Mickey Mouse</h1>
+                            <p>Multicoloured soft toys Disney</p>
+                            <span class="text-secondary">₹299</span>
+                            <div class="rating-container">
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon fa fa-star"></i>
+                            </div>
+                            <br>
+                            <div class="card-buttons">
+                                <a class="cart-button">Add to Cart</a>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="vertical-card simple-card">
+                        <img src="/assets/toy2.jpg" alt="image">
+                        <div class="card-desc-block">
+                            <h1>Mickey Mouse</h1>
```

Comment:
`h1` should be used exclusively in a webpage, that is only once. Replace this with something like `h2` or `h3`

Reply:
okay!